databricks_cli/jobs/cli.py

```diff
@@ -134,11 +139,16 @@ def get_cli(job_id):
 
 
 @click.command(context_settings=CONTEXT_SETTINGS)
-@click.option('--job-id', required=True, type=int)
-@click.option('--jar-params', default=None)
-@click.option('--notebook-params', default=None)
-@click.option('--python-params', default=None)
-@click.option('--spark-submit-params', default=None)
+@click.option('--job-id', required=True, type=JobIdClickType())
+@click.option('--jar-params', default=None, type=JsonClickType(),
+              help='JSON array of parameters. i.e. ["param1", "param2"]')
```

Comment:
`JSON string specifying an array of parameters`. It is not clear if users need to quote the entire value if we say `JSON array`.